layouts/partials/css.html

```diff
@@ -1,5 +1,18 @@
+{{ $inServerMode := site.IsServer }}
+{{ $sass         := "sass/style.sass" }}
+{{ $cssOutput    := "css/style.css" }}
```

Comment:
hello, @lucperkins. I may have misunderstood, but is `$sass` set to the file `sass/style.sass`, and `sass/style.sass` is being removed as part of this commit? Also, are there any benefits to using postCSS instead?

Reply:
@kbhawkey Here, `sass/style.sass` refers to the file in `assets/sass/style.sass`, which is not being removed. The `assets` directory is what Hugo uses as its reference point for Sass processing, much like it uses `static` for static assets.

As for PostCSS, I'm afraid I don't quite understand the question. As far as I know, PostCSS is not used to convert Sass to CSS.

Reply:
OK, thanks for the info. I am reading up on the sass processing/hugo pipes in the hugo docs and came across postcss-cli. Just wondering if that is a separate processing step, after sass conversion to css, and if it would be worthwhile to consider.

Reply:
@kbhawkey Yes, PostCSS is a separate step and has a variety of plugins for things like cross-browser compatibility, pruning unused CSS classes, etc. Worth exploring in the future but would be a supplement to the changes here.